src/bootstrap/src/core/sanity.rs

```diff
@@ -169,11 +172,7 @@ than building it.
             continue;
         }
 
-        // Some environments don't want or need these tools, such as when testing Miri.
-        // FIXME: it would be better to refactor this code to split necessary setup from pure sanity
-        // checks, and have a regular flag for skipping the latter. Also see
-        // <https://github.com/rust-lang/rust/pull/103569#discussion_r1008741742>.
-        if skip_target_sanity {
+        if skip_target_sanity && target != &build.build {
```

Comment:
Does `target != &build.build` mean "target is the host"?

Reply:
It means target is not host.

Reply:
Seems worth a comment somewhere to explain that we interpret `skip_target_sanity` to mean "skip checks for *non-host* targets".

It's not clear that `build` is the "host" target, that seems to diverge from the usual Rust naming conventions.

Reply:
Fields of `Build` represents these from the configuration:

https://github.com/rust-lang/rust/blob/cdb775cab50311de54ccf3a07b331cc56b0da436/config.example.toml#L184-L210

> Seems worth a comment somewhere to explain that we interpret skip_target_sanity to mean "skip checks for non-host targets".

Sure

Reply:
> Fields of Build represents these from the configuration:

Ah, thanks for the link!